packages/edit-post/src/components/preferences-modal/index.js

```diff
@@ -152,6 +157,15 @@ export default function EditPostPreferencesModal() {
 									label={ __( 'Display block breadcrumbs' ) }
 								/>
 							) }
+							<EnableFeature
+								featureName="linkControlSettingsDrawer"
+								help={ __(
+									`Toggle's default open/closed state of the link creation interface's settings drawer.`
+								) }
```

Comment:
```suggestion
								help={ __(
									`Toggles advanced link creation settings?`
								) }
```

As per https://github.com/WordPress/gutenberg/pull/52321/files#r1255404317